builder/tart/step_run.go

```diff
@@ -207,6 +207,12 @@ func typeBootCommandOverVNC(
 
 	ui.Say("Connected to the VNC!")
 
+	if config.BootWait != 0 {
```

Comment:
I don't see the SDK making any assertion for the value except setting the default 10 seconds if the value is 0. At the same time documentation states to set it to a negative value to disable the behaviour. I think it will be better to do a more strict validation here:

```suggestion
	if config.BootWait > 0 {
```